ops/marketplace-tf/modules/gcp/compute_instances/compute/main.tf

```diff
@@ -74,7 +74,7 @@ locals {
   bacalhau_env_vars = {
     LOG_LEVEL                   = "debug"
     BACALHAU_NODE_LOGGINGMODE   = "default"
-    BACALHAU_DIR                = "/data"
+    BACALHAU_DIR                = "/bacalhau_repo"
```

Comment:
This is okay, but wonder if it might be better in /opt/bacalhau/* eventually ?
 
c.f. https://refspecs.linuxfoundation.org/FHS_3.0/fhs/index.html